packages/client/components/AzureDevOpsScopingSearchResults.tsx

```diff
@@ -99,16 +98,6 @@ const AzureDevOpsScopingSearchResults = (props: Props) => {
   const usedServiceTaskIds = useGetUsedServiceTaskIds(estimatePhase)
   const handleAddIssueClick = () => setIsEditing(true)
 
-  const getProjectId = (url: URL) => {
-    const firstIndex = url.pathname.indexOf('/', 1)
-    const seconedIndex = url.pathname.indexOf('/', firstIndex + 1)
```

Comment:
+1 tiny typo `secondIndex`

Reply:
That's in the removed code?

Reply:
Yep, ignore! Will get myself a coffee...